bundles/org.openhab.binding.lghombot/README.md

```diff
@@ -0,0 +1,95 @@
+# LG HomBot Binding
+
+The binding integrates hacked LG HomBot VR6260 based vacuum robots.
+Details on how to modify your HomBot can be [found here](https://www.roboter-forum.com/index.php?thread/10009-lg-hombot-3-0-wlan-kamera-steuerung-per-weboberfläche/).
```

Comment:
I'm not such a fan of links containing just something like `here`, better describe what can be found there as part of the link.

Reply:
Is "rooter-forum.com" instead of "found here" good?

Reply:
Yes